debian/xenial/kubeadm/debian/postinst

```diff
@@ -19,6 +19,11 @@ set -o nounset
 
 case "$1" in
     configure)
+        # because kubeadm package adds kubelet drop-ins, we must daemon-reload
+        # and restart kubelet now. restarting kubelet is ok because kubelet
+        # postinst configure step auto-starts it.
+        systemctl daemon-reload
```

Comment:
It should if systemd is doing the crash loop. Doing a restart is the "more right thing" though.